styles/components/common/pop-menu/_core.scss

```diff
@@ -0,0 +1,7 @@
+// Hidden focus tracker on all poppers
+// Ensure that it never messes up layout
+// Cannot do visibility:hidden or display:none or focus won't trigger
+.hidden-focus-tracker {
```

Comment:
oh looks like I got rid of the need for this and never deleted it.... doh

Reply:
Ok, updated and removed them!